### PR TITLE
Improve AccountHash explanation

### PR DESCRIPTION
**Pull request by andrzej-casper**
### What does this PR introduce?

Polished version of types pages, mainly better explanation of _AccountHash_.

### Additional context

These changes are backported from the default branch of [docs-new](https://github.com/casper-network/docs-new) repository.

Original PR authored by @jonas089: https://github.com/casper-network/docs-new/pull/15.

### Checklist

- [x] I ran the docs locally using `yarn install` and `yarn run start`.
- [x] All links (internal and external) have been verified.
- [x] All technical procedures have been tested.

### CODE REVIEW

File: source/docs/casper/developers/json-rpc/types_chain.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Required Parameters:
 
 ## AccountHash {#accounthash}
 
-Hex-encoded Account hash.
+The AccountHash is a Hex-encoding of the PublicKey and therefore can be viewed as an address.
```
Comment (ipopescu) [inline source/docs/casper/developers/json-rpc/types_chain.md:23]: We have been cautious about the term `address` in the documentation. See issues https://github.com/casper-network/docs/issues/404, https://github.com/casper-network/docs/issues/95. Also, see this definition in the [docs](https://docs.casperlabs.io/concepts/accounts-and-keys/).
```suggestion
The AccountHash is a 32-byte hash derived from a supported PublicKey. Its role is to standardize keys that can vary in length.
```

**Review by ipopescu — CHANGES_REQUESTED** [review]

**Review by ipopescu — APPROVED** [review]
Thank you, @jonas089!

File: source/docs/casper/developers/json-rpc/types_chain.md (hunk 1 above)
Comment (ipopescu) [reply source/docs/casper/developers/json-rpc/types_chain.md:23]: Please take a look. I am not sure why this was resolved. I don't see a new commit.

**Comment by jonas089** [comment]
Thanks, @ipopescu.
Now everything should be committed.

**Review by ipopescu — APPROVED** [review]
LGTM!

**Pull request merged**